dev-env/bin/daml-sdk-head

```diff
@@ -58,7 +64,21 @@ if [[ -d "$DAML_HOME" && "$NUKE" -ne 0 ]]; then
   rm -rf "$DAML_HOME"
 fi
 
-readonly DAML_HEAD_SDK="$DAML_HOME/sdk/0.0.0"
+sha_version() {
+    local dirty date nr sha8
+    dirty=$([[ -z $(git status -s) ]] && echo clean || echo dirty)
```

Comment:
The way I’ve usually seen these kinds of version numbers build up is to include `dirty` if it’s dirty and nothing if it’s not dirty. Any reason why we want `clean` in there explicitly?

Reply:
If you want a truly meaningful version number, you may want to hash the tree if it's dirty. spider-on-x does this.

1. [hash-repository.sh](https://github.com/DACH-NY/spider-on-x/blob/master/helpers/hash-repository.sh)
2. [hash-tree.sh](https://github.com/DACH-NY/spider-on-x/blob/master/helpers/hash-tree.sh)

Reply:
Not really, no. I'll update.